jenkins/job-configs/kubernetes-jenkins/kubernetes-build.yaml

```diff
@@ -6,6 +6,7 @@
         - build-discarder:
             num-to-keep: 200
     builders:
+        - activate-gce-service-account
```

Comment:
need to make sure `gs://kubernetes-release-dev` is writable (probably isn't right now)